### PR TITLE
Readme updates

### PR DESCRIPTION
**Pull request by jenweber**
Add a little more info to help someone new get started.

@sivakumar-kailasam let me know if you want anything else added.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,9 +1,36 @@
...
-This app is used to produce versioned Ember api docs.  To view the data generator producing its data, please visit https://github.com/ember-learn/ember-jsonapi-docs
+This app is used to produce versioned [Ember api docs](https://emberjs.com/api).  To view the data generator producing its data, please visit https://github.com/ember-learn/ember-jsonapi-docs
 
 ## Want to help out?
-We'd love to get help working on the [next set of issues we're seeing](https://github.com/ember-learn/ember-api-docs/projects/2) & to see how to get started on this project, checkout [CONTRIBUTING.md](https://github.com/ember-learn/ember-api-docs/blob/master/CONTRIBUTING.md)
+
+We'd love to get help working on the [next set of issues we're seeing](https://github.com/ember-learn/ember-api-docs/projects/2).
+
+If you're a new contributor, come say hi in the Ember Community Slack's
+team-learning channel, and look for the tags "Good for New Contributors"
```
Comment (sivakumar-kailasam) [inline README.md]: May be we can link to the labels here?

**Comment by toddjordan** [comment]
❤️

**Comment by jenweber** [comment]
@sivakumar-kailasam I added a link for the labels.

**Comment by sivakumar-kailasam** [comment]
Thanks @jenweber 👍

**Pull request merged**